includes/wcpdf-functions.php

```diff
@@ -341,3 +341,23 @@ function wcpdf_convert_encoding( $string, $tool = 'mb_convert_encoding' ) {
 	
 	return $string;
 }
+
+/**
+ * Safe redirect or die.
+ *
+ * @param  string          $url
+ * @param  string|WP_Error $message
+ * @return void
+ */
+function wcpdf_safe_redirect_or_die( $url = '', $message = '' ) {
+	if ( ! empty( $url ) ) {
+		wp_safe_redirect( $url );
```

Comment:
I was testing to see if it redirected to Google. But then I understood it allows only the domain specified as Site Address (URL) in settings.
Just wanted to ask do you think users may wish to redirect to other domains out of their website.

Reply:
I don't think so.